styles/signals.mapcss

```diff
@@ -79,10 +79,10 @@ way["railway:etcs"]["railway:etcs"!=no]["railway:etcs"!=0].tracks
 	z-index: 10;
 	color: blue;
 }
-way["railway:atb"=yes]["railway:etcs"!=yes]["railway:etcs"!=1]["railway:etcs"!=2]["railway:etcs"!=3].tracks
-way["railway:atb-eg"=yes]["railway:etcs"!=yes]["railway:etcs"!=1]["railway:etcs"!=2]["railway:etcs"!=3].tracks
-way["railway:atb-ng"=yes]["railway:etcs"!=yes]["railway:etcs"!=1]["railway:etcs"!=2]["railway:etcs"!=3].tracks
-way["railway:atb-vv"=yes]["railway:etcs"!=yes]["railway:etcs"!=1]["railway:etcs"!=2]["railway:etcs"!=3].tracks
+way["railway:atb"=yes][!"railway:etcs"]["railway:etcs"=no]["railway:etcs"=0].tracks
```

Comment:
You need 2 rules for each etcs check: it can only be either empty or "no", but not both at the same time. But this can all be handled much easier (sorry for misleading you with my first comments): remove all checks for ETCS. Put your rules _before_ the etcs-block (i.e. move it one block up), so ETCS would overwrite any previously set system.

Reply:
No problem at all. I hope that the new update is correct